test/services/xccdf_report_parser_test.rb

```diff
@@ -220,17 +221,17 @@ class TestParser < ::XccdfReportParser
         benchmark: @report_parser.benchmark
       ).save(validate: false)
 
+      assert_equal 367, @report_parser.op_rules.count
       assert_difference('Rule.count', 365) do
         @report_parser.save_rules
       end
 
-      @report_parser.rules = nil
-
+      # All rules already exist so none are saved
       assert_difference('Rule.count', 0) do
         @report_parser.save_rules
       end
 
-      assert_equal @report_parser.op_rules.count, @report_parser.rules.count
+      assert_equal @report_parser.op_rules.count - 2, @report_parser.rules.count
```

Comment:
There are 2 pre-existing rules, so the total number of rules we import, and the total number of rules in general is `@report_parser.rules` - 365